src/index.js

```diff
@@ -29,18 +29,27 @@ export default class ReactPasswordStrength extends React.Component {
     });
   }
 
-  checkValidity(score, length) {
-    const { minScore, minLength } = this.props;
-    return score >= minScore && length >= minLength;
+  tooShort(password) {
+    const { minLength } = this.props;
+    return minLength && password.length < minLength;
```

Comment:
hmmm, this time i disagree. `checkLength` isn't meaningful either. how about `isTooShort`? because it is what it does and returns a boolean. `checkLength` does not say what it returns.